### ISSUE TITLE
Error connecting to sensor os.. + Calling config service failed

### ISSUE THREAD

**Issue by learnhard22** [ouster-lidar/ouster_example#422]
Hey,

since yesterday I can't launch my OS0 anymore. I actually changed nothing to the days before, but the sensor was connected to another laptop but only for reading data, so I can't imagine that this is a problem at all.
I checked the hostname, I changed the udp_dest to the ouster one, to the computer one and left it blank to set it automatically (got it from another issue here) but nothing works.

This is my output when I launch the sensor with "roslaunch ouster_ros ouster.launch"

... logging to /home/stapler1/.ros/log/b6c8d554-260f-11ed-86ca-e15f0be99969/roslaunch-stapler1-stud-22579.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://stapler1-stud:41651/

SUMMARY
========

PARAMETERS
 * /os_cloud_node/tf_prefix: 
 * /os_node/imu_port: 7503
 * /os_node/lidar_mode: 1024x10
 * /os_node/lidar_port: 7502
 * /os_node/metadata: metadata.zwischen...
 * /os_node/replay: False
 * /os_node/sensor_hostname: os-992214001102.l...
 * /os_node/timestamp_mode: TIME_FROM_PTP_1588
 * /os_node/udp_dest: 
 * /os_node/udp_profile_lidar: 
 * /rosdistro: noetic
 * /rosversion: 1.15.14

NODES
  /
    os_cloud_node (ouster_ros/os_cloud_node)
    os_node (ouster_ros/os_node)
    relay_imu (topic_tools/relay)
    relay_lidar (topic_tools/relay)

ROS_MASTER_URI=http://localhost:11311

process[os_node-1]: started with pid [22594]
process[os_cloud_node-2]: started with pid [22595]
[ INFO] [1661610452.442946898]: Client version: 0.4.0+43107a1-dirty-release
process[relay_lidar-3]: started with pid [22601]
[ INFO] [1661610452.446817860]: Starting sensor os-992214001102.local initialization...
[ INFO] [1661610452.447148623]: Will use automatic UDP destination
process[relay_imu-4]: started with pid [22607]
[ERROR] [1661610583.234217686]: Error connecting to sensor os-992214001102.local
[ERROR] [1661610583.237303303]: os_cloud_node: Calling config service failed
================================================================================REQUIRED process [os_node-1] has died!
process has died [pid 22594, exit code 1, cmd /home/stapler1/ROS1_Ouster_SLAM/devel/lib/ouster_ros/os_node __name:=os_node __log:=/home/stapler1/.ros/log/b6c8d554-260f-11ed-86ca-e15f0be99969/os_node-1.log].
log file: /home/stapler1/.ros/log/b6c8d554-260f-11ed-86ca-e15f0be99969/os_node-1*.log
Initiating shutdown!
================================================================================
[relay_lidar-3] killing on exit
[relay_imu-4] killing on exit
[os_cloud_node-2] killing on exit
[os_node-1] killing on exit
shutting down processing monitor...
... shutting down processing monitor complete
done
------------------------------------------------------------------------------------------------
When I ping the sensor I get this result: (I had to cancel the process because nothing happened after the first line)

ping os-992214001102.local
PING os-992214001102.local (192.168.1.201) 56(84) bytes of data.
^C
--- os-992214001102.local ping statistics ---
8 packets transmitted, 0 received, 100% packet loss, time 7149ms

**Comment by kairenw** [comment]
Hi @learnhard22,

The 100% packet loss with your sensor means that for some reason the sensor and your computer simply aren't communicating. I'd try your other laptop again; powering on/off the sensor and your computer, checking the ethernet connection to your computer, and checking out the results of `ifconfig`. Sorry to be so vague but it could be a variety of issues.

For more hands on support you can also contact Ouster support [here](https://ouster.atlassian.net/servicedesk/customer/portal/8).

Closing now as this is not related to the function of the SDK code.
